test "should handle nested conditionals" failing

Hi, if i run `npm run test` on freshly cloned master, One test fails:

```
SUMMARY:
✔ 94 tests completed
✖ 1 test failed

FAILED TESTS:
  Object types
    ✖ should handle nested conditionals
      Node.js (darwin; U; rv:v5.6.0)
    AssertionError: expected 'stats.count must be greater than or equal to 5' to include 'must be at least 5'
        at originalMethod (/Users/giacomo/workspace/yup/tests-webpack.js:6867:70 <- webpack:///~/chai-as-promised/lib/chai-as-promised.js:276:0)
        at doAsserterAsyncAndAddThen (/Users/giacomo/workspace/yup/tests-webpack.js:6889:34 <- webpack:///~/chai-as-promised/lib/chai-as-promised.js:298:0)
        at Assertion.addChainableMethod.originalGetter (/Users/giacomo/workspace/yup/tests-webpack.js:6869:26 <- webpack:///~/chai-as-promised/lib/chai-as-promised.js:278:0)
        at assert (/Users/giacomo/workspace/yup/tests-webpack.js:3828:50 <- webpack:///~/chai/lib/chai/utils/addChainableMethod.js:83:0)
        at /Users/giacomo/workspace/yup/tests-webpack.js:11215:29 <- webpack:///test/object.js:445:31
        at tryCallOne (/Users/giacomo/workspace/yup/tests-webpack.js:8438:13 <- webpack:///~/promise/lib/core.js:37:0)
        at /Users/giacomo/workspace/yup/tests-webpack.js:8504:16 <- webpack:///~/promise/lib/core.js:103:0
        at flush (/Users/giacomo/workspace/yup/tests-webpack.js:7608:30 <- webpack:///~/asap/browser-raw.js:52:0)
        at handleTimer (/Users/giacomo/workspace/yup/tests-webpack.js:7761:14 <- webpack:///~/asap/browser-raw.js:205:0)
        at Timer.listOnTimeout (timers.js:92:15)
```

Have you considered using travis? Thanks!

--- a/test/object.js
+++ b/test/object.js
@@ -442,7 +442,7 @@ describe('Object types', function(){
 
       inst.validate({ stats: { isBig: true, count: 3 }, other: true }).should.be.rejected
         .then(function(err){
-          err.errors[0].should.contain('must be at least 5')
+          err.errors[0].should.contain('must be greater than or equal to 5')
         }),
 
       inst.validate({ stats: { isBig: true, count: 10 }, other: true }).should.be.fulfilled

